app/utils/client_id.json

```diff
@@ -0,0 +1,10 @@
+{
+  "installed":{
+    "client_id":"458262568495-4qna9lg1e5dis43vau696il709gvufvs.apps.googleusercontent.com",
+    "project_id":"tech-committee-calendar",
+    "auth_uri":"https://accounts.google.com/o/oauth2/auth",
+    "token_uri":"https://oauth2.googleapis.com/token",
+    "auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs",
+    "client_secret":"ws9Ejwd_-rbXk0zuvKXH58xY"
+  }
+}
```

Comment:
this is non-sensitive information, and client secret can be and will be, revoked on the server side

Reply:
@Johan-Cuevas opening a git task to close this down and use environment variables